package.json

```diff
@@ -23,7 +23,7 @@
     "scripts/*"
   ],
   "scripts": {
-    "dev": "lerna run dev --parallel --ignore demo-*",
+    "dev": "lerna run dev --parallel --scope '@mux-elements/*'",
```

Comment:
`--scope` is sort of the opposite of `--ignore` where only matching packages are included. Only the main packages are prefixed with `@mux-elements/*` (and the `/` here is important, otherwise lerna doesn't find them).
I swapped them because it was easier to reason and explain how to include other packages in the README_INTERNAL.md file, since to add demos you would add `--scope 'demo-*'` rather than removing things or swapping things out.

Reply:
Cool I'm down for that as long as it's functionally equivalent with our current setup.

Reply:
Yup, it's equivalent!